### ISSUE TITLE
MorphDialogToFab animation glitch

### ISSUE THREAD

**Issue by lgvalle** [nickbutcher/plaid#37]
This has been driving me mad for a couple of days now:

According to the code, when dismissing the dialog there should be an animation to hide dialog child views:

```
        // hide child views (offset down & fade out)
        if (endValues.view instanceof ViewGroup) {
            ViewGroup vg = (ViewGroup) endValues.view;
            for (int i = 0; i < vg.getChildCount(); i++) {
                View v = vg.getChildAt(i);
                v.animate()
                        .alpha(0f)
                        .translationY(v.getHeight() / 3)
                        .setStartDelay(0L)
                        .setDuration(50L)
                        .setInterpolator(AnimationUtils.loadInterpolator(vg.getContext(),
                                android.R.interpolator.fast_out_linear_in))
                        .start();
            }
        }
```

But as you can see in this slow motion capture it never happens. It just goes blank.

![plaid](https://cloud.githubusercontent.com/assets/354372/11046093/349b1944-8723-11e5-9fcb-1fd640ccdede.gif)

If this custom transition is removed (so the default one is executed) then you can see the dialog content animated back into the fab. 
So there must be something in `ChangeBounds` removing hero view children from the animation. But I can't find what it is.

**Comment by jaredrummler** [comment]
The duration of the animation is 0.05 second (50 milliseconds).

**Comment by nickbutcher** [comment]
Fixed by 0d32dcbcea265385873e02ab1c72509b5f4b7ea2
